cmd/influx/write.go

```diff
@@ -68,94 +62,133 @@ or add an entire file specified with an @ prefix.`
 		},
 	}
 	opts.mustRegister(cmd)
-
+	cmd.PersistentFlags().StringVar(&writeFlags.Format, "format", "", "Input format, either lp (Line Protocol) or csv (Comma Separated Values). Defaults to lp unless '.csv' extension")
+	cmd.PersistentFlags().StringVarP(&writeFlags.File, "file", "f", "", "The path to the file to import")
+
+	cmdDryRun := opt.newCmd("dryrun", func(command *cobra.Command, args []string) error {
```

Comment:
can you break out the `dryrun` command into its own command?

Reply:
Sure